src/server.ts

```diff
@@ -54,11 +55,15 @@ app.post('/mobile/:exchangeUserId/transfer', async (req: any, res: any) => {
   await mobileService.transferFunds(req, res)
 })
 
-app.post('/webhook/backup', async (req: any, res: any) => {
+app.post('/webhook/fetch', authMiddleware, async (req: any, res: any) => {
```

Comment:
oh yeah i was tripping when I was saying its all the same endpoint if we're using different paths. idk how i missed that. 

this all looks good for now, we can figured out webhook best practices later.